bind9/README.md

```diff
@@ -1,77 +1,229 @@
+    ```
+    logging {
+     channel default_log {
+          file "/var/log/named/query.log" versions 3 size 10m;
+          print-time iso8601-utc;
+          print-category yes;
+          print-severity yes;
+     };
+       category default { default_log; };
+    }
+    ```
+3. Save and exit the file.
+4. Restart the service
+    ```
+    service named restart
+    ```
+
+#### Syslog
+1. Log in to your Bind9 device.
+2. Open `named.conf` file to add a logging clause:
+    ```
+    logging {
+     channel <example_channel> {
+          syslog <syslog_facility>;
+          severity (critical | error | warning | notice | info | debug [level ] | dynamic);
+          print-time (yes | local | iso8601 | iso8601-utc);
+          print-category yes;
+          print-severity yes;
+     };
+     category <example-category> { <example_channel>; };
+    }
+    ```
+    **NOTE**: Recommended value for `print-time` is `iso8601-utc` because Datadog expects all logs to be in the UTC time zone by default. If the timezone of your Bind9 logs is not UTC please make sure to follow [these](#timezone-steps) steps. Also, check the categories defined by Bind9 [here][16].
+    
+    Example logging channel:
+    ```
+    logging {
+     channel default_log {
+          syslog local3;
+          print-time iso8601-utc;
+          print-category yes;
+          print-severity yes;
+     };
+       category default { default_log; };
+    }
+    ```
+
+3. Save and exit the file.
+4. Edit the syslog/rsyslog configuration to log to your Datadog using the facility you selected in Bind9:
+    ```
+    <syslog_facility>.* @@<DATADOG_AGENT_IP_ADDRESS>:<PORT>
+    ```
+5. Restart the following services.
+    ```
+    service syslog/rsyslog restart
+    service named restart
+    ```
+
+**Note**: Make sure you have mentioned `print-category` and `print-severity` as yes in the channels configured for Bind9 application.
 
 ### Configuration
 
+#### Metric collection
+
 1. Edit the `bind9.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][7] to start collecting your Bind9 [metrics](#metrics). See the [sample bind9.d/conf.yaml][8] for all available configuration options.
 
    ```yaml
    init_config:
-
+  
    instances:
      - url: "<BIND_9_STATS_URL>"
    ```
 
 2. [Restart the Agent][9]
 
+#### Log collection
+
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in the `datadog.yaml` file:
+
+   ```yaml
+   logs_enabled: true
+   ```
+
+#### File Monitoring
+
+1. Add this configuration block to your `bind9.d/conf.yaml` file to start collecting your Bind9 logs.
+
+   See the [sample bind9.d/conf.yaml][8] for available configuration options.
+
+   ```yaml
+   logs:
+     - type: file
+       path: /var/log/named/*.log
+       service: bind9
+       source: bind9
+   ```
+   **Note**: Change the `path` variable in `conf.yaml` to the same as configured in `file` parameter in channels for Bind9 application.
```

Comment:
```suggestion
   **Note**: Change the `path` variable in `conf.yaml` to the same path configured in the `file` parameter in channels for Bind9 application.
```
Should the last part be "for the Bind9 application"?

Reply:
Yes, Sure